### PR TITLE
324 manual capture

### PR DESCRIPTION
**Pull request by ahmetoz**
Adds manual capture documentation.

### CODE REVIEW

File: extension/docs/ManualCapture.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+The `manualCaptureResponse` contains a PSP reference associated with this manual capture request.
+
+``` json
+{
+   "pspReference":"8825408195409505",
+   "response":"[capture-received]"
+} 
+```
+
+CTP payment representation:
```
Comment (lojzatran) [inline extension/docs/ManualCapture.md:38]: Do you want to add that a transaction is created in this case?

Comment (ahmetoz) [reply extension/docs/ManualCapture.md:38]: Make sense, I will add.

Comment (ahmetoz) [reply extension/docs/ManualCapture.md:55]: https://github.com/commercetools/commercetools-adyen-integration/pull/336/commits/aeb7c7dd508e9d850b6f3d2ed80d281c37d3d67f

Comment (ahmetoz) [reply extension/docs/ManualCapture.md:55]: https://github.com/commercetools/commercetools-adyen-integration/pull/336/commits/59fe010df330592b41acaecc1d8e2441e6169635

**Pull request merged**